server/protocols/jmap/src/test/java/org/apache/james/jmap/model/UpdateMessagePatchTest.java

```diff
@@ -80,7 +86,116 @@ public void applyStateShouldReturnSeenWhenPatchSetsSeenOnSeenMessage() {
     }
 
     @Test
-    public void isIdentityShouldReturnTrueWhenNoFlags() {
+    public void applyStateShouldThrowWhenBothIsFlagAndKeywords() {
```

Comment:
Yes, we cannot update state with both is* and keyword and in builder pattern it has that checking